src/unix/fs.c

```diff
@@ -1598,7 +1598,8 @@ static int uv__fs_statx(int fd,
     flags |= AT_SYMLINK_NOFOLLOW;
 
   rc = uv__statx(dirfd, path, flags, mode, &statxbuf);
-
+  uv__msan_unpoison(&statxbuf, sizeof(struct uv__statx));
```

Comment:
Shouldn't this go directly into `uv__statx` right where the actual syscall is being done?